pkgs/development/python-modules/scikitlearn/default.nix

```diff
@@ -3,25 +3,39 @@
 , buildPythonPackage
 , fetchPypi
 , fetchpatch
-, gfortran, glibcLocales
-, numpy, scipy, pytest, pillow
+, gfortran
+, glibcLocales
+, numpy
+, scipy
+, pytest
+, pillow
 , cython
 , joblib
 , llvmPackages
 , threadpoolctl
+, pythonOlder
 }:
 
 buildPythonPackage rec {
   pname = "scikit-learn";
   version = "0.24.1";
   # UnboundLocalError: local variable 'message' referenced before assignment
-  disabled = stdenv.isi686;  # https://github.com/scikit-learn/scikit-learn/issues/5534
+  disabled = pythonOlder "3.6" || stdenv.isi686;  # https://github.com/scikit-learn/scikit-learn/issues/5534
```

Comment:
Can you put the two conditions on separate lines so it's clear which one the link applies to? What is the reason to disable it for Python < 3.6?

Reply:
According https://github.com/scikit-learn/scikit-learn/issues/5534#issuecomment-153104025, we should actually be able to build this on 32bit again.

Reply:
I marked it as disabled on python < 3.6 because of this:

> `Scikit-learn 0.20 was the last version to support Python 2.7 and Python 3.4. Scikit-learn 0.21 supported Python 3.5-3.7. Scikit-learn 0.22 supported Python 3.5-3.8. Scikit-learn now requires Python 3.6 or newer.` 

src: https://scikit-learn.org/stable/install.html

Reply:
:+1: I removed the disabled bit for 32-bit